src/ziggurat/messaging/consumer.clj

```diff
@@ -1,45 +1,33 @@
 (ns ziggurat.messaging.consumer
```

Comment:
We should have integration tests covering the following cases:
1) publish and consume using  proto (expected: success)
2) [Backward Compatibility] publish and consume using `ziggurat.message_payload.MessagePayload` (expected: success and include an assertion on proto deserialization returning a nil)
3) [Backward Compatibility] publish using `ziggurat.message_payload.MessagePayload`  and consume using `ziggurat.mapper.MessagePayload` (expected:success and include an assertion on proto deserialization returning a nil)
4) publish a random String which is neither a protobuf nor a MessagePayload (expected: consume should return a nil)